ragnarok_formats/src/map.rs

```diff
@@ -318,6 +318,12 @@ impl FromBytes for MapResources {
                 ResourceType::LightSource => {
                     let mut light_source = LightSource::from_bytes(byte_stream).trace::<Self>()?;
                     light_source.position.y = -light_source.position.y;
+
+                    // Some light sources have color channels with values bigger than 1.0 (255), so
+                    // we need to clamp them.
+                    // TODO: Does this maybe have a special meaning?
```

Comment:
Yes, if we want to increase the percieved dynamic range and implement an HDR effect in SDR, we need to implement proper tone mapping in our light calculation (for example using Reinhard tone mapping).

Reply:
True, it might be an error in the source files.